experimental/packages/otlp-exporter-base/src/platform/browser/OTLPExporterBrowserBase.ts

```diff
@@ -73,24 +75,14 @@ export abstract class OTLPExporterBrowserBase<
     const body = JSON.stringify(serviceRequest);
 
     const promise = new Promise<void>((resolve, reject) => {
-      if (this._useXHR) {
-        sendWithXhr(
-          body,
-          this.url,
-          this._headers,
-          this.timeoutMillis,
-          resolve,
-          reject
-        );
-      } else {
-        sendWithBeacon(
-          body,
-          this.url,
-          { type: 'application/json' },
-          resolve,
-          reject
-        );
-      }
+      sendWithXhr(
+        body,
+        this.url,
+        this._headers,
+        this.timeoutMillis,
+        resolve,
+        reject
+      );
```

Comment:
This is a breaking change. Beacon sending is an important feature that can't be removed without significant reasoning and discussion

Reply:
This is an environment specific requirement and there is precedent for breaking from spec for such reasons. Without beacon sending telemetry can be lost in many cases.

Reply:
I do know the advantages of `sendBeacon`. But the `user-agent` header is required. The `sendBeacon` function will never be reached.

Reply:
Using `fetch` with `keepalive=true` also allows the request to outlive the page. If we can replace `xhr` with `fetch`.

Reply:
`fetch` with keep-alive is not supported in all environments, so we can't rely on simply using it instead.
While (most) current versions of environments that support `sendBeacon` also support fetch keep alive, my understanding is that it is still considered to be expermenital.

Reply:
user-agent is a SHOULD requirement not a MUST requirement: https://github.com/open-telemetry/opentelemetry-specification/blob/ce2c5946faa9c961ea01d89c7aed83621865b67b/specification/protocol/exporter.md#user-agent

Reply:
Further discussion: #3845

Reply:
regardless I think the refactor should be done separately from this PR as it is a very significant change